### PR TITLE
cli: skip Example_demo_locality

### PR DESCRIPTION
**Pull request by knz**
Informs #99903.

Release note: None

### CODE REVIEW

**Review by ajwerner — APPROVED** [review]
LGTM

File: pkg/cli/demo_locality_test.go [hunk 1]
```diff
@@ -14,12 +14,19 @@
...
+	"fmt"
+
 	"github.com/cockroachdb/cockroach/pkg/cli/democluster"
 	"github.com/cockroachdb/cockroach/pkg/security/securityassets"
 	"github.com/cockroachdb/cockroach/pkg/util/log"
 )
 
 func Example_demo_locality() {
+	_ = disabledExample_demo_locality
+	fmt.Println("skipped due to flake: https://github.com/cockroachdb/cockroach/issues/99903")
```
Comment (dt) [inline pkg/cli/demo_locality_test.go:26]: is this better than just renaming the func `skippedExample....` ?

**Comment by knz** [comment]
Merging as per cockroachlabs.atlassian.net/browse/CREQ-839

File: pkg/cli/demo_locality_test.go (hunk 1 above)
Comment (knz) [reply pkg/cli/demo_locality_test.go:26]: lol i only saw your message afterwards. sorry about that

Comment (dt) [reply pkg/cli/demo_locality_test.go:26]: I dismissed it right after I posted -- decided it wasn't worth waiting for.

**Comment by knz** [comment]
blathers backport 23.1 23.1.0

**Pull request merged**